MyAmazingRPG/MyAmazingRPG/src/Main.java

```diff
@@ -0,0 +1,54 @@
+//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Main {
+    public static void main(String[] args) {
+
+        Character broccoli = new Character();
+
+        broccoli.name = "Super Broccoli";
+        broccoli.health = 50;
+        broccoli.strength = 20;
+        broccoli.defense = 30;
+        broccoli.damage = 0;
+        broccoli.isAlive = true;
+
+        Character spinach = new Character();
+
+        spinach.name = "Magic Spinach";
+        spinach.health = 50;
+        spinach.strength = 30;
+        spinach.defense = 10;
+        spinach.damage = 0;
+        spinach.isAlive = true;
+
+        do {
+            spinach.attack(broccoli);
+            broccoli.takeDamage(spinach.damage);
```

Comment:
might be redundant as when spinach attacks broccoli -> broccoli can take damage automatically during an attack